### PR TITLE
add SELU to Tensor

### PR DESCRIPTION
**Pull request by geohotstan**
was prepping an onnx_ops clean up and found a missed op from onnx_ops

### CODE REVIEW

File: tinygrad/tensor.py [hunk 1]
```diff
@@ -2688,6 +2688,19 @@ def celu(self, alpha=1.0):
...
+    Applies the Scaled Exponential Linear Unit (SELU) function element-wise.
+
+    - Described: https://paperswithcode.com/method/selu
+    - Paper: https://arxiv.org/abs/1706.02515v5
+
+    ```python exec="true" source="above" session="tensor" result="python"
+    print(Tensor([-3., -2., -1., 0., 1., 2., 3.]).softsign().numpy())
+    ```
+    """
+    return scale * (self.relu() - (-alpha * self.exp() + alpha).relu())
```
Comment (geohotstan) [inline tinygrad/tensor.py:2702]: `return scale * (self > 0).detach().where(self, alpha * (self.exp() - 1))`
which implementation is cleaner? ^ This one is slightly faster and looks more like the implementation in `https://paperswithcode.com/method/selu`

File: tinygrad/tensor.py [hunk 2]
```diff
@@ -2688,6 +2688,19 @@ def celu(self, alpha=1.0):
     """
     return self.maximum(0) + (alpha * ((self / alpha).exp() - 1)).minimum(0)
 
+  def selu(self, alpha=1.67326, scale=1.0507):
```
Comment (chenyuxyz) [inline tinygrad/tensor.py:2691]: just use alpha and gamma? it matches onnx naming and one less indirection

File: tinygrad/tensor.py (hunk 1 above)
Comment (chenyuxyz) [reply tinygrad/tensor.py:2702]: yea i think `scale * (self >= 0).detach().where(self, alpha * (self.exp() - 1))` is more readable

File: tinygrad/tensor.py [hunk 3]
```diff
@@ -2688,6 +2688,19 @@ def celu(self, alpha=1.0):
...
+    """
+    Applies the Scaled Exponential Linear Unit (SELU) function element-wise.
+
+    - Described: https://paperswithcode.com/method/selu
+    - Paper: https://arxiv.org/abs/1706.02515v5
+
+    ```python exec="true" source="above" session="tensor" result="python"
+    print(Tensor([-3., -2., -1., 0., 1., 2., 3.]).softsign().numpy())
```
Comment (chenyuxyz) [inline tinygrad/tensor.py:2699]: softsign?

File: tinygrad/tensor.py (hunk 2 above)
Comment (geohotstan) [reply tinygrad/tensor.py:2691]: image

pytorch called it `scale` but I guess people just call `.selu()` anyways 
will change to gamma
